pkg/jx/cmd/step_create_task.go

```diff
@@ -394,7 +390,7 @@ func (o *StepCreateTaskOptions) generatePipeline(languageName string, pipelineCo
 			name = kube.ToValidName(shortName)
 		}
 
-		pipeline, tasks, structure, err := lifecycles.Pipeline.GenerateCRDs(name, o.buildNumber, "will-be-replaced", "abcd", o.PodTemplates, taskParams)
```

Comment:
nit: I think the part of the TODO up on line 385 related to namespaces (`Create client now to ... namespace when applying rather than during generation?`) is done now that the namespace passed to `GenerateCRDs` is real, and I think you can delete [this line](https://github.com/jenkins-x/jx/blob/ea547624f85ecdbae9c573dc4a91f8e39b076879/pkg/jx/cmd/step_create_task.go#L821) and maybe [this one](https://github.com/jenkins-x/jx/blob/ea547624f85ecdbae9c573dc4a91f8e39b076879/pkg/jx/cmd/step_create_task.go#L832) as well.